R/pdf_document.R

```diff
@@ -201,7 +202,7 @@ pdf_document <- function(toc = FALSE,
 # Use filter to set pdf geometry defaults (while making sure we don't override
 # any geometry settings already specified by the user)
 pdf_pre_processor <- function(metadata, input_file, runtime, knit_meta, files_dir,
-                              output_dir) {
+                              output_dir, extra_dependencies) {
```

Comment:
Okay, I didn't notice that it was a separate function. I thought it was inside `pdf_document()` like the pre-processor for `html_document()`